movies/src/main/java/com/fundacionjala/movies/MovieChildren.java

```diff
@@ -0,0 +1,35 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by Charito on 6/10/2016.
+ */
+public class MovieChildren extends Movie {
+    public static final int CHILDREN = 2;
+
+    private static final double PRICE_RENT = 1.5;
```

Comment:
If you notice the MovieChildren, MovieNewRelease, MovieRegular classes, you are repeating the PRICE_RENT, DELAY, PRICE_DELAY variables in all those classes.
Please abstract those variable into the Movie class